app/controllers/citizens/additional_accounts_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Citizens
+  class AdditionalAccountsController < BaseController
+    def index; end
+
+    def create
+      case params[:additional_account]
+      when 'yes'
+        redirect_to new_citizens_additional_account_path
+      when 'no'
+        # TODO: - set redirect path when known
+        render plain: 'Landing page: Next step in Citizen journey'
+      else
+        flash[:error] = 'You must select either Yes or No'
```

Comment:
**[suggestion]** If we were to handle this submission as normal forms that need to be validated, we could leverage out of the box from the form validation errors, the already existent way to display the form errors if it's not valid and all this copy would be set in the locales, reducing this action code to something like the following:

```ruby
def create
  @form = Citizens::AdditionalAccountsForm.new(form_params)
  if @form.valid?
     redirect_to new_citizens_additional_account_path
  else
    render :new
  end  
end
```

Reply:
I'd agree with you if this wasn't such a simple case - in that there is only one error condition (submit is clicked without choosing either Yes or No).

Reply:
Let's agree to disagree on this :bowtie: , a lot of the custom code, including the case statements in the controller, the custom error display, the bespoke copy set directly in the views, could just be managed automatically by driving the form submission through the conventional Rails model validations/locales.